src/components/tasks/card/index.tsx

```diff
@@ -313,11 +323,20 @@ const Card: FC<CardProps> = ({
         const data = {
             percentCompleted: percentCompleted,
         };
-        await updateTasks({
-            task: data,
-            id: id,
-        });
-        toast(SUCCESS, 'Progress Updated Successfully');
+        if (isUserAuthorized) {
+            await updateTask({
```

Comment:
here we're using await, but at the same time `.then()` is also being used

Reply:
below is the reason mentioned